### ISSUE TITLE
Comment Replies Lost in Conversions

### ISSUE THREAD

**Issue by MyKo101** [jgm/pandoc#6959]
If I have a `.docx` file and add a comment, then convert it to `.md`, the result is something along the lines of:

```
[This is the comment]{.comment-start id="1" author="Mike" date="2020-12-17T16:53:00Z"} This is the content being commented on[]{.comment-end id="1"}
```

Here it is again with some new lines inserted to make it a bit easier to read (note it will not compile with these newlines)
```
[This is the comment]
  {.comment-start id="1" author="Mike" date="2020-12-17T16:53:00Z"}
This is the content being commented on
[]
  {.comment-end id="1"}
```

Which can then be converted back into `.docx` by pandoc and the file remains unchanged (the comment is still there).

However, if I add a reply to that comment (common when collaborating), the resulting `.md` contains:
```
[This is the comment]{.comment-start id="1" author="Mike" date="2020-12-17T16:53:00Z"}[Here is my reply]{.comment-start id="2" author="Mike" date="2020-12-17T17:39:00Z"} This is the content being commented on[[]{.comment-end id="2"}]{.comment-end id="1"}
```

Again, it's easier to read if I add newlines:
```
[This is the comment]
  {.comment-start id="1" author="Mike" date="2020-12-17T16:53:00Z"}
[Here is my reply]
  {.comment-start id="2" author="Mike" date="2020-12-17T17:39:00Z"}
This is the content being commented on
[
  []
    {.comment-end id="2"}
]
  {.comment-end id="1"}
```

So the `.comment-start` of the reply comment comes immediately after the `.comment-start` of the original comment. And the `.comment-end` of the reply comes in the `[]` brackets of the `.comment-end` of the original comment.

However, when I re-compile this `.md`, I lose that reply comment. When I convert it back to `.md` again, the reply comment is gone completely.

Is this deliberate behaviour from pandoc, is it an issue or am I doing something wrong here?

I have tried wrapping the comment and content in the reply comment, and all this does is add a second comment on the same content, but *not* a reply.

**Comment by jgm** [comment]
@jkr may be able to shed light.

**Comment by mb21** [comment]
So when converting the docx to md all the content (including comments) is still there, but when you go back from that md to docx it's lost?

**Comment by MyKo101** [comment]
Exactly. They no longer show in the `.docx` file, and then if I convert that back into `.md`, they have gone entirely

**Comment by mmouniro** [comment]
Bumped into this issue. Is there any progress or workaround ? @MyKo101

**Comment by jgm** [comment]
Same issue as #8189.
